instrumentation/jetty-8.0/src/main/java/io/opentelemetry/auto/instrumentation/jetty/JettyHandlerInstrumentation.java

```diff
@@ -53,11 +53,11 @@ public JettyHandlerInstrumentation() {
   @Override
   public String[] helperClassNames() {
     return new String[] {
-      packageName + ".JettyHttpServerTracer",
+      "io.opentelemetry.instrumentation.servlet.HttpServletRequestGetter",
```

Comment:
Can you add a comment, at least "Order matters here"?

Reply:
yes, excellent point, done